refarch-gateway/pom.xml

```diff
@@ -214,4 +217,23 @@
             </plugin>
         </plugins>
     </build>
+
+    <scm>
+        <url>https://github.com/it-at-m/refarch.git</url>
+        <connection>scm:git:https://github.com/it-at-m/refarch.git</connection>
+        <developerConnection>scm:git:https://github.com/it-at-m/refarch.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+    <licenses>
+        <license>
+            <name>MIT</name>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <organization>it@M</organization>
+            <email>opensource@muenchen.de</email>
+            <url>https://github.com/it-at-m</url>
+        </developer>
+    </developers>
```

Comment:
Should we add this block to refarch-templates as well? This way when people create a new project, they already have the metadata in place to build maven artifacts

Reply:
Yes good idea, but we just first verify if this is enough. Couldn't find any definition which fields are exactly needed.

Reply:
Ok, I will create a issue in refarch-templates. :)